packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -67,15 +86,6 @@ export async function processOutboundDocumentRetrievalResps({
       requestId,
       source: MedicalDataSource.CAREQUALITY,
     });
-
-    await setDocQueryProgress({
-      patient: { id: patientId, cxId: cxId },
-      downloadProgress: {
-        status: "completed",
-      },
-      requestId,
-      source: MedicalDataSource.CAREQUALITY,
-    });
```

Comment:
This was removed because the tally should be the one that sets the progress to completed, or failed if the counts align